### ISSUE TITLE
Typo in curl command

### ISSUE THREAD

**Issue by yonirab** [argoproj/argo-workflows#3546]
Checklist:

 * [ ] I've included the version.
 * [ ] I've included reproduction steps.
 * [ ] I've included the workflow YAML.
 * [ ] I've included the logs.
 
**What happened**:
Following instructions in https://github.com/argoproj/argo/blob/master/docs/access-token.md, when running the `curl` commands I get 
```
{"error":"no valid authentication methods found for mode server","code":16,"message":"no valid authentication methods found for mode server"}
```
**What you expected to happen**:
Should have worked per the doc.
**How to reproduce it (as minimally and precisely as possible)**:
Follow the doc.
**Anything else we need to know?**:
Yes!! There appears to be a typo in the doc.
It works fine with "Authorization: $ARGO_TOKEN" instead of "Authorisation: $ARGO_TOKEN" (spelt with a "z" instead of an "s").
Not sure if this might be a locale issue?

**Message from the maintainers**:

If you are impacted by this bug please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.

**Comment by simster7** [comment]
Would you like to open a PR with these fixes? 🙂

**Comment by alexec** [comment]
Available for testing in [v2.10.0-rc1](https://github.com/argoproj/argo/releases/tag/v2.11.0-rc1).
